ActindoWorkFlow/UserInterface/03a_Processes.md

```diff
@@ -0,0 +1,634 @@
+
+
+The list displays all tokens if the ![Toggle](/Assets/Icons/Toggle.png "[Toggle]") *Hide processed* toggle is disabled. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+  - *Status*  
+  Token status. The different statuses are displayed in different colours. The following statuses are possible:
+
+    - *Ready*
+    - *Processed*
+    - *Being processed*
+    - *Error*
+
+[comment]: <> (Tokens kann auch als "Ready", "Being processed", "Error" im Status vorkommen. So gemeint oder nur Processed/Unprocessed? Add colour code?)
+
+  - *Place*  
+  Place where the token has been input.
+
+  - *Data*  
+  Type of data contained in the token.
+
+  - *Next possible action(s)*    
+  Next compatible action(s) where the token can be input.
+
+  - *ID*  
+  Token ID number.  
+
+  [comment]: <> (Oli fragen)
+
+  - *Place ID*  
+  Place ID number.
+
+  - *Process Action ID*  
+  Process action ID number.
+
+  [comment]: <> (Oli fragen. Null wenn am end place?)
+
+
+
+## Create view
+
+*Workflows > Processes > Tab OVERVIEW > Button Points > Menu entry create*
+
+![Create view](/Assets/Screenshots/ActindoWorkFlow/Workflows/CreateView.png "[Create view]")
+
+
+- *Name*  
+Enter a name for the view.
+
+- [CANCEL]  
+Click this button to cancel creating a view. The *Create view* window is closed.
+
+- [SAVE]  
+Click this button to save the new view. The *Create view* window is closed. The view is saved and displayed in the *View* drop-down list.
+
+
+
+## Rename view
+
+*Workflows > Processes > Tab OVERVIEW > Button Points > Menu entry rename*
+
+![Rename view](/Assets/Screenshots/ActindoWorkFlow/Workflows/RenameView.png "[Rename view]")
+
+
+- *Name*  
+Click the field to edit the view name.
+
+- [CANCEL]  
+Click this button to cancel renaming a view. The *Rename view* window is closed.
+
+- [SAVE]  
+Click this button to save the changes to the view name. The *Rename view* window is closed. The new name for the view is saved and displayed in the *View* drop-down list.
+
+
+
+## Log ID
+
+*Workflows > Processes > Tab OVERVIEW > Select a process > Tab Logs > Select Log Message*
+
+![Log ID](/Assets/Screenshots/ActindoWorkFlow/Processes/LogID.png "[Log ID]")
+
+- ![Back](/Assets/Icons/Back02.png "[Back]") (Back)  
+Click this button to close the *Log ID* view and return to the *Logs* tab.
+
+- *Log ID (x)*  
+Log ID number.
+
+- *Log Message*  
+A description of the issue is displayed in the text box. This box is read-only.  
+
+  > [Info] In case of an error, detailed information is provided (stack trace) for reporting and debugging purposes.
+
+[comment]: <> (Diesen Text kann man rauskopieren, um ein Ticket zu erstellen o.ä. Erwähnen? Nützlich für Kunden?)
+
+- *Log Level*  
+Description of log level. The following levels are available:
+
+  - *Debug*
+  - *Info*
+  - *Notice*
+  - *Warning*
+  - *Error*
+  - *Critical*
+  - *Alert*
+  - *Emergency*
+
+
```

Comment:
```suggestion
- Log message box   
  Detailed log message with further information to the log type. 
  
  > [Info] In case of an error, detailed information is provided (stack trace) for reporting and debugging purposes. 
```